### PR TITLE
[LPN-900] migrate address on new France cities system

### PR DESCRIPTION
**Pull request by elvino29**

### CODE REVIEW

File: tests/Entity/PostAddressTest.php [hunk 1]
```diff
@@ -3,13 +3,16 @@
...
 
 use App\Entity\PostAddress;
-use PHPUnit\Framework\TestCase;
+use Tests\App\AbstractKernelTestCase;
+use Tests\App\TestHelperTrait;
 
-class PostAddressTest extends TestCase
+class PostAddressTest extends AbstractKernelTestCase
 {
+    use TestHelperTrait;
```
Comment (ottaviano) [inline tests/Entity/PostAddressTest.php:11]: it's already imported in AbstractKernelTestCase

File: tests/Entity/OAuth/UserAuthorizationTest.php [hunk 2]
```diff
@@ -5,14 +5,16 @@
...
 use Ramsey\Uuid\Uuid;
 use Ramsey\Uuid\UuidInterface;
+use Tests\App\AbstractKernelTestCase;
+use Tests\App\TestHelperTrait;
 
-class UserAuthorizationTest extends TestCase
+class UserAuthorizationTest extends AbstractKernelTestCase
 {
+    use TestHelperTrait;
+
```
Comment (ottaviano) [inline tests/Entity/OAuth/UserAuthorizationTest.php:17]: it's already imported in AbstractKernelTestCase
```suggestion
```

File: tests/Geo/ZoneMatcherTest.php [hunk 3]
```diff
@@ -6,9 +6,12 @@
 use App\Entity\PostAddress;
 use App\Geo\ZoneMatcher;
 use Tests\App\AbstractKernelTestCase;
+use Tests\App\TestHelperTrait;
 
 class ZoneMatcherTest extends AbstractKernelTestCase
 {
+    use TestHelperTrait;
```
Comment (ottaviano) [inline tests/Geo/ZoneMatcherTest.php:13]: same here

File: src/Doctrine/Hydrators/EventHydrator.php [hunk 4]
```diff
@@ -111,6 +112,6 @@ private function createForeignAddress(
         ?float $latitude,
         ?float $longitude
     ): PostAddress {
-        return PostAddress::createForeignAddress($country ?? '', $zipCode ?? '', $cityName, $street ?? '', $latitude, $longitude);
+        return PostAddress::createForeignAddress($country ?? '', $zipCode ?? '', $cityName, $street ?? '', null, $latitude, $longitude);
```
[the remark this answers is not in the document]
Comment (elvino29) [reply src/Doctrine/Hydrators/EventHydrator.php:115]: to handle Region value. We have an issue here. The latitude value is set instead of Region value

**Pull request merged**